### DISCUSSION TITLE
Getting ModuleNotFoundError: No module named 'semantic_kernel.connectors.ai.function_choice_behavior'

### DISCUSSION THREAD

**Discussion by awonglk** [microsoft/semantic-kernel#7168] [Q&A] [answered]
I'm attempting to run the example here:
python/samples/concepts/plugins/openai_function_calling_with_custom_plugin.py

Have definitely run:
`pip install semantic-kernel
`
However, when I run the sample, I'm getting:
ModuleNotFoundError: No module named 'semantic_kernel.connectors.ai.function_choice_behavior'

Isn't this part of "semantic-kernel" package? Or do I need to import something else?

**Comment by sophialagerkranspandey** [comment, marked as answer]
We are in the process of getting this release out. You should see this released shortly. Thanks!

**Comment by moonbox3** [comment]
SK Python 1.2.0 is now available, which contains this new `FunctionChoiceBehavior`.
